index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hijab Center</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Caramel&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Covered+By+Your+Grace&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="brand">
+      <h1 class="brand-name">Taiba Hijab Center</h1>
+      <p class="discription">THANE'S PREMIUM HIJAB CENTER</p>
+    </div> 
+    <input type="checkbox" id="openside">
+    <label for="openside" class="sidebaricon">
+      <div class="spinner top"></div>
+      <div class="spinner mid"></div>
+      <div class="spinner bottom"></div>
+    </label>
+    <nav class="sidebar">
+      <ul class="menu">
+        <li><a href="#">NEW ARRIVALS</a></li>
+        <li><a href="#">HIJAB</a></li>
+        <li><a href="#">DRESESS</a></li>
+        <li><a href="#">ACCESSORIES</a></li>
+        <li><a href="#">ABOUT</a></li>
+      </ul>
+      <ul class="social-network">
+        <li><a href="#" class="insta"><img src="./images/1.png" alt="insta"></a></li>
+        <li><a href="#" class="fb"><img src="./images/2.png" alt="facebook"></a></li>
+      </ul>
+    </nav>
+   </div>
+  </header>
+  <article class="slideshow">
+    <div class="slider">
+      <div class="slide one"><button>NEW ARRIVALS</button></div>
+      <div class="slide two"></div>
+      <div class="slide three"></div>
+      <div class="slide four"><button>SHOP NOW</button></div>
+    </div>
```

Comment:
Add the first slide element before the ending so it will get an infinite scrolling effect. IF you are still confused checkout my Image Slider repo in my github profile.
```suggestion
    <div class="slide one"><button>NEW ARRIVALS</button></div>
    </div>
```

Reply:
yes i did it as u have told me to that, i go through from your repo for my better understanding of it... now my slide is perfectly woking in infinite scrolling effect.